### ISSUE TITLE
notochord part of axial skeletal system?

### ISSUE THREAD

**Issue by alanruttenberg** [obophenotype/uberon#1459]
I was looking at notochord and see:

1. notochord part of axial skeletal system
2. axial skeletal system has part skeletal joint

But the notochord exists before there are any skeletal joints, right?

http://purl.obolibrary.org/obo/UBERON_0002328
http://purl.obolibrary.org/obo/UBERON_0011137
https://en.wikipedia.org/wiki/Notochord#/media/File:Gray19_with_color.png

**Comment by balhoff** [comment]
I bet [amphioxus](https://en.wikipedia.org/wiki/Branchiostoma_lanceolatum) don't have any skeletal joints.

**Comment by ukemi** [comment]
Ascidians too.

**Comment by RDruzinsky** [comment]
If intervertebral discs and cartilage are considered parts of skeletal
systems then the notochord should certainly be part of the skeletal system.

**Comment by cmungall** [comment]
yes, ivdisk is part of the skeletal system:
![image](https://user-images.githubusercontent.com/50745/46928538-e7ecf680-cfef-11e8-9964-17f4fefe3850.png)

@RDruzinsky @wdahdul we decided on the current definition of skeletal tissue way back in #134 but I still think it could do with tightening:

_A specialized form of connective tissue in which the extracellular matrix is firm, providing the tissue with resilience, and/or mineralized and that functions in mechanical and structural support_ [VSAO]" [VSAO:0000015, GO_REF:0000034, http://dx.plos.org/10.1371/journal.pone.0051070, PSPUB:0000170, https://github.com/obophenotype/uberon/issues/134]

Not a fan of the and/or construct. Is this just saying it may be mineralized, it doesn't have to be?

In that case, isn't invertebrate chitin (even fungal chitin) skeletal? What about keratin based tissue? As I recall we wanted to restrict this to a certain enumerated set of tissues (e.g. bone, cartilage, dentine, and enamel) but I can't find record of this?

**Comment by RDruzinsky** [comment]
This is a bit odd. Intervertebral discs are not cartilage. And, remnants of
the embryonic notochord remain as part of the adult disc. I think this
needs work.

**Comment by cmungall** [comment]
* A cartilage organ in FMA (which uberon follows)
 * Wikipedia says "Each disc forms a cartilaginous joint"
 * connective tissue in MA
 * "fibrocartilaginous articulation" various other sources
 * Spongy discs located between the vertebrae of the spinal column; composed of the outer annulus fibrosus and inner nucleus pulposus. [NCIT:C49571]
 * "A pad of fibrocartilage between" uberon text def, from Kardong

but if we agree that it is composed of 2 parts better to start with those than trying to classify mixed tissue

nucleus pulposus

 * "the jelly-like substance in the middle of the spinal disc which is a remnant of the notochord" [MP:0006392] (mmm, jelly) (this is the current uberon def)
 * mucoid tissue in FMA

annulus fibrosis

 * "A ring of fibrous or fibrocartilaginous tissue (as of an intervertebral disk or surrounding an orifice of the heart)." (uberon def)
 * fibrocartilage

this is our opportunity to better here, let's avoid text like "jelly-like". It seems to be an organ composed of two tissue types, mucoid + fibrocartilage. (hardly makes sense to call the combo pure cartilage).

**Comment by RDruzinsky** [comment]
Sorry for the typo. I should have said that Intervertebral discs are not
just cartilage.

I think that we have to have a better sense of what we mean by skeletal
system. If bone, cartilage, and intervertebral discs are parts of a
skeletal parts then why exclude the notochord? Perhaps instead of skeletal
system we could call it "internal skeletal system" or "internal supporting
system" is_a "supporting system." Then an exoskeleton could be is_a
"external supporting system." I suppose that the problem would be that
these are, to some extent, definitions based on function. But skeletal
system, which includes so many tissue types, has the same problem, right?

Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
druzinsk@uic.edu

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky

**Comment by cmungall** [comment]
The skeletal system is the skeleton plus the articular system, and the IDs would be included in the latter - so it's not quite analogous to the notochord.

**Comment by RDruzinsky** [comment]
Okay, but parts of the notochord do become parts of the intervertebral
discs in the adult. And, what do we do with replacement cartilages in the
developing embryo? Are they not parts of some skeletal system, even before
any formation of joints?


Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
druzinsk@uic.edu

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky

**Comment by cmungall** [comment]
Yes, the ontogenic start time should be made explicit. I'd say yes to including cartilage elements (otherwise you end up saying sharks have no/little skeleton). But exclude mesenchymal precursors?

**Comment by RDruzinsky** [comment]
Vertebral elements derive from mesenchyme/mesoderm, so are you suggesting
that the notochord be excluded because it is a mesenchymal precursor? I'm
not sure where to draw the line? Are parts of the somites part of the
skeletal system? Difficult problem. I think that the notochord is part of
the skeletal system. It certainly acts as a skeletal structure in some
chordates and it forms the nucleus pulposus in adult vertebrates.


Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
druzinsk@uic.edu

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky

**Comment by cmungall** [comment]
just to be clear I don't have anything against the notochord (in fact I love em), just trying to achieve consensus on the overall framework from which we can then make principled decisions about various elements.

My thinking was that somites would not be part of the skeletal system (but they are obviously precursors).

Agree that both notochord is functionally skeletal and developmental precursor to skeletal system, but the problem with either of these criteria is that you open the doors to on the one hand hydrostatic skeletons and on the other somites, various mesenchymal blobbings... neither of which is necessarily wrong, but important to be aware of the consequences..

**Comment by alanruttenberg** [comment]
@RDruzinsky, you write "If intervertebral discs and cartilage are considered parts of skeletal
systems then the notochord should certainly be part of the skeletal system."

That's not my read. At issue is that when the notochord starts existing as compared to the skeleton/skeletal system or to the discs and cartilage. A part relation holds over the whole time the notochord exists (as a notochord). See [Relations in biomedical ontologies](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r46).

I'm not an expert in anatomy or development, so please tell me if I'm wrong. It doesn't look like there's more than a wish of a skeletal system in humans when the notochord first appears. That would seem to indicate that there should *not* be an assertion: notochord partof axial skeletal system.

**Comment by tgbugs** [comment]
To generalize from the point of view of tissues for a moment. If intervertebral cartilage is considered part of the skeletal system, then are the parts of my nose and ears that are made of cartilage also skeletal? This seems to defy common sense (excuse the term) logic about what constitute a skeleton. That is not to say that it defies what a 'skeletal system' is, but at that point, 'skeletal system' becomes a weasel word that is arbitrary and not based on any consistent properties of its constituents.

To bring up a couple of related perspectives.
Cartilage has a radically different set of disorders than the skeleton.  
The basic physical properties of the tissue are fundamentally distinct.  
Can things that do not exist at the same time be part of each other?  
What we defined the skeleton as tissue that can be mineralized during fossilization under certain conditions?

In brief. I agree with @alanruttenberg and that statement from @RDruzinsky indicates that if anything intervertebral discs and cartilage should not be considered part of the skeletal system.

Which leads to some deeper questions.
Why were joints included in the skeletal system in the first place?  
Would we not be better served by making joints distinct and creating a joint-skeletal-system?
How are ligaments currently treated? (Answer: skeletal ligaments are part of the skeletal system, but ligaments are not. Suggesting that 'skeletal cartilage' might be created as another way to box cartilage that is part of the skeletal system.)

**Comment by RDruzinsky** [comment]
@alanruttenberg I see your point that a developmental precursor, such as
mesoderm or mesenchyme would not be a part of an adult skeletal system, but
those are multipotent structures that develop into many diverse tissues in
the adult. The notochord is a supporting structure in the embryo that
remains a supporting structure as the nucleus pulposus in the
adult. @tgbugs So, the os penis (UBERON:0006435) or baculum is part of the
skeletal system in Uberon. I assume that, historically, it is there only
because it is a bone. I would argue that the notochord is more a skeletal
element than the os penis and that if the notochord is not a skeletal
element then the os penis is not.


Robert E. Druzinsky, Ph.D.
Clinical Associate Professor
Dept. of Oral Biology
College of Dentistry
University of Illinois at Chicago
801 S. Paulina
Chicago, IL 60612
druzinsk@uic.edu

Office: 312-996-0406
Lab: 312-996-0629
Website: www.peerj.com/RobertDruzinsky
